Date.php

```diff
@@ -5455,7 +5453,7 @@ function getStandardPartSecond()
      * @static
      * @since    Method available since Release 1.5.0
      */
-    function _addOffset($pn_offset,
+    static function _addOffset($pn_offset,
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.